apps/system/js/wallpaper_manager.js

```diff
@@ -349,6 +349,10 @@
       // Create a new blob:// url for this blob
       this._blobURL = URL.createObjectURL(blob);
 
+      document.getElementById('screen').style.backgroundImage =
+        'linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)),' +
+        'url(' + this._blobURL + ')';
+
```

Comment:
This one is moved from the original bootstrap.js to assign the blob into the wallpaper background image.